### PR TITLE
LPS-174356 Button to delete (Trash icon) fields on the Action card not works

### PR DESCRIPTION
**Pull request by manuelecastro**
Ticket: [LPS-174356](https://issues.liferay.com/browse/LPS-174356)

Also fixes: [LPS-178201](https://issues.liferay.com/browse/LPS-178201)

Previous PR: https://github.com/liferay-objects/liferay-portal/pull/1716

### CODE REVIEW

**Comment by manuelecastro** [comment]
ci:stop

**Comment by manuelecastro** [comment]
ci:test:sf

**Comment by manuelecastro** [comment]
ci:test:relevant

**Comment by manuelecastro** [comment]
ci:test:object

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/PredefinedValuesTable.tsx [hunk 1]
```diff
@@ -198,7 +200,7 @@ export default function PredefinedValuesTable({
 		};
 
 		const deletePredefinedValueField = ({itemData}: {itemData: Item}) => {
-			const [name] = itemData.name.props.children;
+			const name = itemData.name;
```
Comment (danvb112) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/PredefinedValuesTable.tsx:203]: Hey @manuelecastro,you can use [Destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) here, we always use it here when it is possible to make the code simpler and more readable, like: `const {name} = itemData;`

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @manuelecastro, good job on this bug! I have just a quick JavaScript suggestion :smile:

**Comment by manuelecastro** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/PredefinedValuesTable.tsx (hunk 1 above)
Comment (manuelecastro) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/PredefinedValuesTable.tsx]: Hi! Just commited the change you asked :) it really makes the code simpler by doing the destructuring assignment, thanks for the review!

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by danvb112** [comment]
ci:forward

**Pull request closed without merging**